syntax = "proto3";

package api.unsafe.hackingdevice;

service HackingDeviceServer {
  rpc read_secret_station_data(Void) returns (SecretStationDataResponse) {}
}

message SecretStationDataResponse { repeated string data = 1; }
message Void {}